tests/functional/test_import.py

```diff
@@ -172,13 +172,6 @@ def test_import_non_existing_raises_import_error(pyi_builder):
             raise RuntimeError("ImportError not raised")
         """)
 
-# :todo: Use some package which is already installed for some other
-# reason instead of `simplejson` which is only used here.
-@importorskip('simplejson')
```

Comment:
What do you think about keeping this for Python2? Of course then the Py3-specific code can be removed from the test-script.

Reply:
Good point. I've kept this test, but now it only runs on Python 2.